.github/workflows/unix.yml

```diff
@@ -7,24 +7,30 @@ jobs:
   build:
 
     runs-on: macos-latest
-    
+
     steps:
     - uses: actions/checkout@v1
 
     - name: Install clang-format
       run: brew install clang-format
 
+    - name: Install cmake-format
+      run: pip install cmake_format
+
     - name: Configure
-      run: cmake -Htest -Bbuild
+      run: cmake -Htest -Bbuild -DFORMAT_CHECK_CMAKE=True
 
     - name: Run format
       run: "cmake --build build --target format"
 
-    - name: Check format
-      run: "! cmake --build build --target check-format"
+    - name: Check clang-format
+      run: "! cmake --build build --target check-clang-format"
+
+    - name: Check cmake-format
+      run: "! cmake --build build --target check-cmake-format"
```

Comment:
It does? `check-cmake-format` wasn't being tested, because, I assume, the step ends at the first error, considering that there's no error output for `cmake-format`. Do you still want me to only test `check-format`?

Reply:
Ah I see what you mean. Still, I'd like to confirm that `check-format` still works as inteded (we only check after formatting later), so adding test for `check-format` would be great imo.

Reply:
That's true.